getting_started/first_3d_game/01.game_setup.rst

```diff
@@ -74,33 +74,25 @@ reliable to block even fast-moving objects.
 
 A box's wireframe appears in the viewport with three orange dots. You can click
 and drag these to edit the shape's extents interactively. We can also precisely
-set the size in the inspector. Click on the *BoxShape3D* to expand the resource.
-Set its *Extents* to ``30`` on the X axis, ``1`` for the Y axis, and ``30`` for
+set the size in the inspector. Click on the :ref:`BoxShape3D <class_BoxShape3D>` to expand the resource.
+Set its *Size* to ``60`` on the X axis, ``2`` for the Y axis, and ``60`` for
 the Z axis.
 
-.. image:: img/01.game_setup/09.box_extents.webp
+.. image:: img/01.game_setup/09.box_size.webp
 
-.. note::
-
-    In 3D, translation and size units are in meters. The box's total size is
-    twice its extents: ``60`` by ``60`` meters on the ground plane and ``2``
-    units tall. The ground plane is defined by the X and Z axes, while the Y
-    axis represents the height.
```

Comment:
What's the purpose of removing this note?

Reply:
In the previous docs you had to set `extents` in this step of the tutorial instead of size. Therefore both mismatched (size being 30x30, extends being 60) which I suppose warranted this explanatory note to explain the difference.

Since users don't have to deal with extents vs. size at this point, this explanation doesn't not make sense anymore (atleast in this tutorial, not sure if extents are used or explained somewhere else).

Reply:
Thanks!